cron/alerts/main.go

```diff
@@ -224,10 +230,6 @@ func makeSESMessage(charSet, subject, msg string) sesTypes.Message {
 }
 
 func (a *Alert) logLastError(lastError string, badRecipients []string) {
-	if lastError == "" {
-		return
-	}
```

Comment:
I'm torn about this change, keeping this makes it simpler to call, but I'm fine either way.

Reply:
Logically and semantically it made sense. There's more refactoring that could be done to help here, but I cut myself off.